### DISCUSSION TITLE
Opcache crashes with PHP 8.2 and Pimcore 11

### DISCUSSION THREAD

**Discussion by bitfactory-henno-schooljan** [pimcore/pimcore#15819] [Q&A / Support]
When using PHP 8.2 and Pimcore 11 when Opcache is enabled, we are seeing 100% CPU usage in the PHP process when we visit a page related to a data object of the `EcommerceFrameworkBundle` and then go back to another page (e.g. the site index page). After that, all new PHP child processes will keep using 100% CPU, not respond anymore, and eventually crash with error code 124 until PHP itself is restarted completely.

After turning off Opcache the issue goes away.

It seems to be related to this, the symptoms are exactly the same:
https://github.com/php/php-src/issues/11736

Has anyone encountered this issue as well, with this bundle or with another, and/or knows a workaround without disabling Opcache entirely?

**Comment by jdreesen** [comment]
Is it enough to deactivate the JIT, or does the complete OPcache have to be deactivated for the problem to disappear?

**Comment by bitfactory-henno-schooljan** [comment]
Unfortunately no, also with `opcache.jit=disabled` the issue persists. Setting `opcache.enable=0` or not loading the extension altogether is needed to make the issue disappear.
